tests/ballerina-test/src/test/java/org/ballerinalang/test/connectors/sql/SQLActionsTest.java

```diff
@@ -169,7 +165,7 @@ public void testSelectIntFloatData() {
         Assert.assertEquals(doubleVal.floatValue(), doubleExpected);
     }
 
-    @Test(groups = "ConnectorTest")
+    @Test(groups = {"ConnectorTest", "H2NotSupported"})
```

Comment:
It is possible to use constants for values "ConnectorTest" and "H2NotSupported" as it is used frequently